Jun_Youngsang/js/config.js

```diff
@@ -0,0 +1,445 @@
+<h3> About <a href="https://geospatial-usace.opendata.arcgis.com/maps/fc0f38c5a19a46dbacd92f2fb823ef8c/about">Military Installations, Ranges, and Training Areas (MIRTA) Data</a> </h3>
+<p>The DoD installations are commonly referred to as a base, camp, post, station, yard, center, homeport facility for any ship, or other activity under the jurisdiction, custody, control of the DoD. Publicly releasable locations of DoD Sites in the 50 states, Puerto Rico, and Guam available through data.gov. This dataset was created from source data provided by the four Military Service Component headquarters and was compiled by the Defense Installation Spatial Data Infrastructure (DISDI) Program within the Office of the Deputy Under Secretary of Defense for Installations and Environment, Business Enterprise Integration Directorate. Sites were selected from the 2009 Base Structure Report (BSR), a summary of the DoD Real Property Inventory.</p>
+`;
+
+const divChapter2 =`
+<h3>West Point (New York)</h3>
+<iframe src="https://www.youtube.com/embed/zw97_4VTu6c?autoplay=1" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+<p>West Point is the oldest continuously occupied regular army post in the U.S. West Point had a major role in U.S. history during the American Revolution. General George Washington considered West Point the most important strategic location in America and in 1778 selected Thaddeus Kosciuszko, a Polish engineer, to design the fortifications for West Point. Washington later transferred his headquarters near West Point in 1779. American Continental Line soldiers constructed forts, gun batteries, and redoubts and installed a 65-ton iron chain across the Hudson River to block British invasions along the main transportation waterway. Since 1802, the <a href="https://www.westpoint.edu/">United States Military Academy</a> has been located.</p>
+`;
+
+const divChapter3 =`
+<h3>Eglin Air Force Base (Florida)</h3>
+<img src="data/images/240801-F-KO637-1614.JPG">
```

Comment:
**Suggestion:** Ensure `<img>` elements have alternate text for web accessibility.

```suggestion
<img src="data/images/240801-F-KO637-1614.JPG"  alt="Describe your image">